.travis.yml

```diff
@@ -33,10 +33,15 @@ script:
   - if [[ "$TRAVIS_GO_VERSION" == 1.12* && "$TRAVIS_OS_NAME" == "osx" ]]; then export GOFLAGS=-mod=vendor; fi
   - go test -tags ci ./...
   - go vet -tags ci -unsafeptr=false ./...
-  - golint -set_exit_status $(go list -tags ci ./...) || exit 1
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi'
-  - go test -tags ci -covermode=atomic -coverprofile=coverage.out ./...
-  - $(go env GOPATH | awk 'BEGIN{FS=":"} {print $1}')/bin/goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
+  - golint -set_exit_status $(go list -tags ci ./...)
+  - |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi
+    set -e
+    go test -tags ci -covermode=atomic -coverprofile=coverage.out ./...
```

Comment:
This might as well go in the if too, no point running a coverage check with no reporting to be done (it's the same tests as above)